torch/_inductor/scheduler.py

```diff
@@ -1930,6 +1930,9 @@ def __init__(self, nodes: list[ir.Operation]) -> None:
         with dynamo_timed("Scheduler.__init__"):
             self._init(nodes)
 
+    # Disable CUBIN saving when measuring speedups.  Saving to disk introduces arbitrary
+    # slowdowns that bias benchmark results.
+    @config.patch("triton.store_cubin", False)
```

Comment:
Nice catch! As we discussed offline, let's find a better place to apply this.

Reply:
@desertfire I added this context manager around the two places in `graph.py` that create `Scheduler` objects, and it seems to work just fine. I've tested that CUBIN is still saved off at some stage in the process if the user requests it. Please give it a look!